src/components/home/SelectPDFButton.container.js

```diff
@@ -11,7 +11,9 @@ export const mapDispatchToProps = (dispatch, { createJob }) => ({
         if (createJob) {
             dispatch(jobActions.createJob(file))
         } else {
-            dispatch(filesToValidateActions.addFile(file))
+            const isElectron = window.location.search.includes('electron=true') || window.location.protocol === 'file:' ;
```

Comment:
This will be useful in many places. So let's not calculate this every time, but put this into const in dedicated module. Let's name it `environment.js` and put in `src` directly.